.github/workflows/plugins-jdk14-test.0.yaml

```diff
@@ -53,6 +53,8 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 8
+      - name: Add checkstyle-plugin
+        run: sed -i '/<\/sourceDirectories>/i <sourceDirectory>scenarios\/jdk14-with-gson-scenario<\/sourceDirectory>' test/plugin/pom.xml
```

Comment:
@kezhenxu94 @dmsolr Are you ok about this?

Reply:
> @kezhenxu94 @dmsolr Are you ok about this?

Works for me

Reply:
@vcjmhg Could you change this yaml as a matrix based? In case we forget we hard coded `jdk14-with-gson-scenario` here.

Reply:
ok